pepper-apis/src/main/resources/db-changes/seed/DDP-5102-add-arabic-portuguese-vietnamese.xml

```diff
@@ -0,0 +1,593 @@
+            <column name="language_code_id" valueComputed="(select language_code_id from language_code where iso_language_code = 'vi')"/>
+            <column name="translation_text" value="Hoàn tất"/>
+        </insert>
+        <insert tableName="i18n_activity_instance_status_type_name_trans">
+            <column name="activity_instance_status_type_id"
+                    valueComputed="(select activity_instance_status_type_id from activity_instance_status_type where activity_instance_status_type_code = 'CREATED')"/>
+            <column name="language_code_id" valueComputed="(select language_code_id from language_code where iso_language_code = 'vi')"/>
+            <column name="translation_text" value="Mới"/>
+        </insert>
+        <insert tableName="i18n_activity_instance_status_type_name_trans">
+            <column name="activity_instance_status_type_id"
+                    valueComputed="(select activity_instance_status_type_id from activity_instance_status_type where activity_instance_status_type_code = 'IN_PROGRESS')"/>
+            <column name="language_code_id" valueComputed="(select language_code_id from language_code where iso_language_code = 'vi')"/>
+            <column name="translation_text" value="Đang tiến hành"/>
+        </insert>
+
+
+        <update tableName="i18n_activity_instance_status_type_name_trans">
+            <column name="translation_text" value="Remplis"/>
+            <where>
+                activity_instance_status_type_id = (select activity_instance_status_type_id from activity_instance_status_type where activity_instance_status_type_code = 'COMPLETE')
+                and language_code_id = (select language_code_id from language_code where iso_language_code = 'fr')
+            </where>
+        </update>
+        <update tableName="i18n_activity_instance_status_type_name_trans">
+            <column name="translation_text" value="Nouveaux"/>
+            <where>
+                activity_instance_status_type_id = (select activity_instance_status_type_id from activity_instance_status_type where activity_instance_status_type_code = 'CREATED')
+                and language_code_id = (select language_code_id from language_code where iso_language_code = 'fr')
+            </where>
+        </update>
+        <update tableName="i18n_activity_instance_status_type_name_trans">
+            <column name="translation_text" value="En cours"/>
+            <where>
+                activity_instance_status_type_id = (select activity_instance_status_type_id from activity_instance_status_type where activity_instance_status_type_code = 'IN_PROGRESS')
+                and language_code_id = (select language_code_id from language_code where iso_language_code = 'fr')
+            </where>
+        </update>
+
+
+        <update tableName="i18n_activity_instance_status_type_name_trans">
+            <column name="translation_text" value="Завершено"/>
```

Comment:
This particular one is NOT from the translators because it didn't seem to make sense. The one I have here what we currently have in database.